assets/scripts/app/MessageBar.jsx

```diff
@@ -67,18 +67,13 @@ class MessageBar extends React.Component {
               {linkText || <FormattedMessage id="msg.more-info" defaultMessage="More info" />}
             </a>
           }
-
-          <button
-            className="close"
+          <CloseButton
             onClick={this.onClickDismiss}
-            title={this.props.intl.formatMessage({ id: 'btn.dismiss', defaultMessage: 'Dismiss' })}
-          >
-            ×
-          </button>
+          />
         </div>
       </Transition>
     )
   }
 }
 
-export default injectIntl(MessageBar)
+export default MessageBar
```

Comment:
Since we don't have a higher-order component anymore let's move the `export default` up to where `MessageBar` is defined on line 12.